tst/EnergyPlus/unit/Psychrometrics.unit.cc

```diff
@@ -433,3 +433,209 @@ TEST_F(EnergyPlusFixture, Psychrometrics_CpAirAverageValue_Test)
+        97.53048245, 97.54950626,  97.56852035,  97.58752557,  97.60651883,  97.62550117,  97.64447612,  97.66343401,  97.68238777,  97.7013245,
+        97.72025336, 97.73917295,  97.75808021,  97.77697826,  97.79586097,  97.81473912,  97.83360053,  97.85245612,  97.87129715,  97.89012997,
+        97.90895465, 97.92776351,  97.94656357,  97.96535557,  97.98413572,  98.00290411,  98.0216607,   98.04040946,  98.0591458,   98.07787434,
+        98.09659062, 98.11529602,  98.13399297,  98.15267543,  98.17135021,  98.19001678,  98.20866815,  98.22731202,  98.24594464,  98.26456913,
+        98.28317933, 98.30178152,  98.32037191,  98.33895574,  98.35752401,  98.37608583,  98.39463599,  98.41317593,  98.43170428,  98.45022567,
+        98.46873238, 98.48723156,  98.50572064,  98.52419839,  98.54266935,  98.56112833,  98.5795743,   98.59801359,  98.61644162,  98.63485926,
+        98.65326612, 98.6716639,   98.69005,     98.70842759,  98.72679422,  98.74515239,  98.76349897,  98.78183652,  98.80016317,  98.818479,
+        98.83678649, 98.85508254,  98.87337098,  98.89164802,  98.90991685,  98.92817432,  98.94642114,  98.96465599,  98.98288655,  99.0011027,
+        99.0193095,  99.03750706,  99.05569661,  99.07387501,  99.09204163,  99.11020164,  99.12834992,  99.14649098,  99.16462101,  99.18274005,
+        99.20084747, 99.21894777,  99.23704034,  99.25511882,  99.2731896,   99.29125019,  99.30930313,  99.32734527,  99.34537664,  99.36340108,
+        99.38141098, 99.39941718,  99.41740889,  99.43539377,  99.45336677,  99.4713336,   99.48928668,  99.50723239,  99.5251714,   99.54309608,
+        99.56101346, 99.57892103,  99.59682135,  99.61470745,  99.63258635,  99.65045553,  99.66831756,  99.68616804,  99.70400948,  99.72184065,
+        99.7396654,  99.75747931,  99.77528304,  99.79307659,  99.81085938,  99.82863649,  99.84640288,  99.8641592,   99.88190929,  99.8996443,
+        99.91737375, 99.93509321,  99.95280653,  99.97050611,  99.98819959,  100.0058826,  100.0235557,  100.041219,   100.0588725,  100.0765193,
+        100.0941565, 100.1117838,  100.1294041,  100.147012,   100.1646135,  100.1822015,  100.1997837,  100.2173562,  100.2349186,  100.2524759,
+        100.2700192, 100.2875567,  100.3050842,  100.3226022,  100.3401102,  100.3576094,  100.3751023,  100.3925827,  100.4100569,  100.4275217,
+        100.4449773, 100.462423,   100.4798599,  100.4972871,  100.514705,   100.532117,   100.5495203,  100.5669127,  100.5842934,  100.6016687,
+        100.6190344, 100.636391,   100.6537386,  100.6710809,  100.6884104,  100.7057304,  100.7230452,  100.7403472,  100.7576436,  100.7749305,
+        100.7922053, 100.809475,   100.8267359,  100.8439874,  100.8612302,  100.8784641,  100.8956893,  100.9129053,  100.9301125,  100.947311,
+        100.9645009, 100.9816821,  100.9988569,  101.0160203,  101.0331752,  101.0503248,  101.067462,   101.0845908,  101.101711,   101.1188265,
+        101.1359293};
```

Comment:
Is this just the same array as inside the code?  I'm not sure I understand what this unit test is doing then.

Reply:
@Myoldmopar  I should have added more details for this unit test. Yes, this sample array `tsat_fn_pb_tsat` is the same sample array for saturated temperature as inside the code `tsat_fn_pb_y`. The purpose of this unit test is to test the correctness of the sample data array. The sample data were extracted from the original psychrometric function `PsyTsatFnPb`  every 64 Pa in the range of 64 Pa to 105,664 Pa. The sample data for saturated temperature from `tsat_fn_pb_tsat` was compared to the results from the original psychrometric function `PsyTsatFnPb`. So we can double-check the correctness of the sample data inside the code.

Reply:
@LipingWang the idea of the unit test is good but instead of repeating the array, we should be testing the array that is created in the header file and used by the interpolation function.

Reply:
@JasonGlazer @Myoldmopar  sounds great!